datasets/bgt/bakken/v1.0.0.json

```diff
@@ -0,0 +1,103 @@
+{
+  "id": "bakken",
+  "title":"bakken",
+  "description": "Object met een permanent karakter dat dient om iets in te bergen of te verzamelen.- Groep: Inrichtingselement - Code: 160",
+  "type": "table",
+  "version": "1.0.0",
+  "crs": "EPSG:28992",
+  "auth": "OPENBAAR",
+  "temporal": {
+    "identifier": "volgnummer",
+    "dimensions": {
+      "geldigOp": [
+        "objectBegintijd",
+        "objectEindtijd"
+      ]
+    }
+  },
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "mainGeometry": "geometrie2D",
+    "identifier": [
+      "identificatie",
+      "volgnummer"
+    ],
+    "required": [
+      "schema",
+      "identificatie"
+    ],
+    "display": "identificatie",
+    "properties": {
+      "schema": {
+        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+      },
+      "bgtType": {
+        "type": "string",
+        "description": "Het soort bak. Bij Bak is dit altijd 'niet-bgt'"
+      },
+      "plusType": {
+        "type": "string",
+        "description": "Het soort bak (IMGeo plus-populatie)."
+      },
+      "geometrie2D": {
+        "$ref": "https://geojson.org/schema/Point.json",
+        "description": "puntgeometrie"
+      },
+      "objectBegintijd": {
+        "type": "string",
+        "description": "Datum waar op het object bij de bronhouder is ontstaan"
+      },
+      "objectEindtijd": {
+        "type": "string",
+        "description": "Datum waarop het object bij de bronhouder niet meer geldig is"
+      },
+      "identificatie": {
+        "type": "string",
+        "description": "Uniek identificerend kenmerk voor het object dat onveranderlijk is zolang het object bestaat"
+      },
+      "volgnummer": {
+        "type": "integer",
+        "description": "Uniek volgnummer van de toestand van het object."
+      },
+      "tijdstipRegistratie": {
+        "type": "string",
```

Comment:
Is dit wellicht een datetime? Dan zou een "format"" "date-time" toegevoegd kunnen worden.

Reply:
ja, ga ik ook aanpassen (generiek, dus duurt wel even)